.github/workflows/ci.yml

```diff
@@ -799,10 +806,6 @@ jobs:
         run: |
           echo "Matrix: $(jq '.' matrix.json)"
           echo "matrix=$(cat matrix.json)" >> $GITHUB_OUTPUT
-      - name: Clean local Maven repo
```

Comment:
If we want to create the cache in this job, we should disable creating it in `maven-checks`.

Would `$MAVEN dependency:go-offline` download test runtime dependencies, like TestNG 5.10?

Reply:
> Goal that resolves all project dependencies, including plugins and reports and their dependencies

It looks like it should.

Reply:
I've already added Maven repo removal at the end of `maven-checks`

Reply:
I don't see changes in `maven-checks` in the diff. We didn't clear the whole local repo, just the build artifacts:

![image](https://user-images.githubusercontent.com/795177/198538046-733c470c-aafd-4c8c-9219-d3470a25ec98.png)

Reply:
> I don't see changes in `maven-checks` in the diff. We didn't clear the whole local repo, just the build artifacts:
> 
> ![image](https://user-images.githubusercontent.com/795177/198538046-733c470c-aafd-4c8c-9219-d3470a25ec98.png)

Oh, I missed that, sorry.

Reply:
Should we add back the part to remove Trino artifacts to this job? Looks like now cache would include Trino artifacts as well? Thanks for catching @nineinchnick.

Reply:
Yes, because we run `mvn install` in this job, because there's a subsequent `mvn validate` which requires Trino artifacts to be in the local repo.